src/widgets/NumericTextCtrl.cpp

```diff
@@ -638,6 +658,8 @@ void NumericTextCtrl::OnMouse(wxMouseEvent &event)
 
       Adjust((int)fabs(steps), steps < 0.0 ? -1 : 1, mFocusedDigit);
       Updated();
+
+      Refresh();
```

Comment:
Why was this new line needed?

Reply:
The real question is how it worked without this line before